.github/workflows/use-deploy-ecs.yml

```diff
@@ -16,15 +16,15 @@ on:
       aws-access-key-id:
         required: true
         description: "AWS_ACCESS_KEY_ID value"
-      aws-role-arn:
-        required: true
-        description: "AWS_ROLE_ARN value"
       aws-secret-access-key:
         required: true
         description: "AWS_SECRET_ACCESS_KEY value"
       docker-repo:
         required: true
         description: "The Docker repository to upload the docker image to, formatted '....amazonaws.com/<app>'"
+      role-to-assume:
```

Comment:
It looks like `aws-role-arn` is the expected value here: https://github.com/mbta/dotcom/pull/1770/files/f0b7e698648251412d857c24aae4a9fe9c9949ff..acb445b39f312cbfee90586b3e09f34cfa824c23#diff-ae570e673d878f853f5159ea079c8f6ae8663fc448b98158726df4762c16596dL104 but you could update it in both places if you wanted.

I think the remaining steps are:
- remove `aws-access-key-id` and `aws-secret-access-key` from the variables here
- remove any use of those values from this file, replacing them with role ARN usage
- remove those values from the workflows which call this one

Reply:
Thanks for these steps--they were a big help, I think I've got it straightened out now